public/apps/configuration/panels/auth-view/instruction-view.tsx

```diff
@@ -35,7 +35,7 @@ export function InstructionView(props: { config: ClientConfigType }) {
       <EuiText textAlign="center" size="xs" color="subdued" className="instruction-text">
         In order to use Security plugin, you must decide on authentication <EuiCode>authc</EuiCode>{' '}
         and authorization backends <EuiCode>authz</EuiCode>. Use{' '}
-        <EuiCode>plugins/opensearch-security/securityconfig/config.yml</EuiCode> to define how to
+        <EuiCode>config/opensearch-security/config.yml</EuiCode> to define how to
```

Comment:
Looks like there is a code formatting issue on this line

Reply:
Should be fixed!